week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -22,18 +22,52 @@ let bottle = {
   volume: 0,
   fillUp: function() {
     // calling this function should pour your bottle full (volume = 100);
+    if(this.volume === 100){
+      console.log("Bottle already full.");
+    }
+    else if(this.volume === 0) {
+      this.volume = this.volume + 100;

```

Comment:
```suggestion
    }  else if (this.volume === 0) {
      this.volume = this.volume + 100;
```
Remember to start `else if` on the same line as the closing curly brace of the `if` statement.